styles/index_style.css

```diff
@@ -0,0 +1,102 @@
+/* 
+Dark Red: #8F0C0C
+Rosewood: #5F1212 
+Dark Sienna: #230707
+Tuscany: #CAA2A2
+Snow: #F7F2F2
+*/
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #230707;
+    background-image: url(/images/cairovercoat-timetiles2.png);
+    background-repeat: repeat;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+a {
+    text-decoration: none;
```

Comment:
Tip: It's considered a best practice (as an accessibility feature) to leave the underline on links so users can find them more easily.